Package.swift

```diff
@@ -1,9 +1,15 @@
 import PackageDescription
 
 let package = Package(
-  name: "SnowMan",
-  dependencies: [
-    .Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,7))
-  ]
+  	name: "SnowMan",
+  	targets: [
+  	  	Target(name: "Dictionary"),
+  		  Target(name: "SnowManLogic"),
+ 		Target(name: "Application", dependencies: ["SnowManLogic", "Dictionary"]),
+ 		Target(name: "Specs", dependencies: ["SnowManLogic", "Dictionary"]),
```

Comment:
maybe break the specs down into 2 additional packages, `SnowManLogicSpec` & `DictionarySpec`